src/main/java/ch/njol/skript/bukkitutil/BukkitUtils.java

```diff
@@ -36,4 +49,22 @@ public static boolean registryExists(String registry) {
 		return null;
 	}
 
+	/**
+	 * Get the inventory slot index of the {@link EquipmentSlot}
+	 * @param equipmentSlot The equipment slot to get the index of
+	 * @return The equipment slot index of the provided slot, otherwise null if invalid
+	 */
+	public static Integer getEquipmentSlotIndex(EquipmentSlot equipmentSlot) {
+		return  BUKKIT_EQUIPMENT_SLOT_INDICES.get(equipmentSlot);
+	}
+
+	/**
+	 * Get the {@link EquipmentSlot} represented by the inventory slot index
+	 * @param slotIndex The index of the equipment slot
+	 * @return The equipment slot the provided slot index, otherwise null if invalid
+	 */
+	public static EquipmentSlot getEquipmentSlotFromIndex(Integer slotIndex) {
```

Comment:
```suggestion
	public static EquipmentSlot getEquipmentSlotFromIndex(int slotIndex) {
```

Reply:
Can you explain why you want this change? The map it's up against is Integer

Reply:
int should be preferred in java wherever possible, same with other primitives.